paper/ms.tex

```diff
@@ -618,9 +604,9 @@ \section{Discussion}
 Neptune-mass planets within 0.3~AU with moderately high eccentricity
 ($e\sim0.1-0.3$) \footnote{
   The two stars were monitored by \project{HARPS}
-  campaign, and it has recently been reported by \citealt{} that HD~20781 hosts
+  campaign, and it has recently been reported by \citealt{todo} that HD~20781 hosts
   four planets between $M\sin(i)\approx 0.006-0.04$~\mjupiter\ with $e \le
-  0.11$ within $\approx 0.35$~AU.}.
+  0.11$ within $\approx 0.35~{\rm AU}$.}.
```

Comment:
For reasons I don't understand, the ~ outside of the math environment here seemed to be causing problems. (Also, I like to keep units in math-mode, but that's just a stylistic thing)